app/controllers/vendor_api/interviews_controller.rb

```diff
@@ -0,0 +1,122 @@
+module VendorAPI
+  class InterviewsController < VendorAPIController
+    include ApplicationDataConcerns
+    include APIValidationsAndErrorHandling
+
+    def create
+      CreateInterview.new(
+        actor: audit_user,
+        application_choice: application_choice,
+        provider: provider_for_interview,
+        date_and_time: interview_params[:date_and_time],
+        location: interview_params[:location],
+        additional_details: interview_params[:additional_details],
+      ).save!
+
+      render_application
+    rescue ProviderAuthorisation::NotAuthorisedError => e
+      render status: :unprocessable_entity, json: {
+        errors: [
+          {
+            error: 'NotAuthorisedError',
+            message: e.message,
+          },
+        ],
+      }
+    rescue ValidationException => e
+      render status: :unprocessable_entity, json: {
+        errors: [
+          {
+            error: 'UnprocessableEntity',
+            message: e.message,
+          },
+        ],
+      }
+    end
+
+    def update
+      UpdateInterview.new(
```

Comment:
discussed in refinement, could we include allowing single field updates in this PR?